.github/workflows/c-fortran-test-hardware.yml

```diff
@@ -7,8 +7,8 @@ jobs:
   test:
     strategy:
       matrix:
-        os: [ubuntu-20.04]
-        compiler: [gcc-9]
+        os: [ubuntu-22.04]
+        compiler: [gcc-11]
         arch: [aarch64, ppc64le]
         distro: [ubuntu20.04]
```

Comment:
Seems weird to use distro 20.04 with os 22.04?

Reply:
As I understand it, the parent container will use 22.04 and run the tests in a child image with 20.04

Reply:
Bumping the run-on-hardware action version let me sync those two